TryIT/settings_global.py

```diff
@@ -4,13 +4,18 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 # MY VARS
-LANDING = True
-READY_FOR_NEW_ED = False
```

Comment:
`READY_FOR_NEW_ED` a bit confusing if only used for the statistics page

Reply:
Solve that issue in the last [commit](https://github.com/DA-ETSIINF/TryIT/pull/71/commits/55b4fc7297c4b425f1f117a9f21b1e407014ea24)